bot/cogs/dm_comms.py

```diff
@@ -0,0 +1,48 @@
+from discord import DMChannel, Message
+from discord.ext import commands
+from rodhaj import Rodhaj
+
+
+class DmCommunications(commands.Cog):
+    def __init__(self, bot: Rodhaj):
+        self._bot = bot
```

Comment:
Not sure why those are marked as private attrs but ok

Reply:
I don't see a reason why there will be calls to dm_comms.bot so I made it private

Reply:
Fair enough